### DISCUSSION TITLE
v5.6.2

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16549] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.6.2'>v5.6.2</a>.</em>

**Comment by s-ohnishi** [comment]
Isn't it `v5.6.2` instead of `v5.6.1`?

**Reply by niden** [reply]
It should have been - not sure what GitHub did there.

**Reply by niden** [reply]
Ok corrected it. Thank you @s-ohnishi
